label_options/caebike-label-options.json

```diff
@@ -1,15 +1,25 @@
 {
   "MODE": [
-    {"value":"walk_run", "baseMode":"WALKING", "met": {"ALL": {"range": [0, -1], "mets": 3}}, "kgCo2PerKm": 0},
-    {"value":"auto_driver", "baseMode":"CAR", "met_equivalent":"IN_VEHICLE", "kgCo2PerKm": 0.22031},
-    {"value":"auto_passenger", "baseMode":"CAR", "met_equivalent":"IN_VEHICLE", "kgCo2PerKm": 0.110155},
-    {"value":"e-bike", "baseMode":"E_BIKE", "met": {"ALL": {"range": [0, -1], "mets": 5}}, "kgCo2PerKm": 0.00728},
-    {"value":"bike", "baseMode":"BICYCLING", "met": {"ALL": {"range": [0, -1], "mets": 6}}, "kgCo2PerKm": 0},
-    {"value":"taxi", "baseMode":"TAXI", "met_equivalent":"IN_VEHICLE", "kgCo2PerKm": 0.30741},
-    {"value":"bike_scooter_share", "baseMode":"BICYCLING", "met": {"ALL": {"range": [0, -1], "mets": 3.5}}, "kgCo2PerKm": 0.00447},
-    {"value":"bus_train", "baseMode":"BUS", "met_equivalent":"IN_VEHICLE", "kgCo2PerKm": 0.179033},
-    {"value":"other_mode", "baseMode":"OTHER", "met_equivalent":"UNKNOWN", "kgCo2PerKm": 0},
-    {"value":"not_a_trip", "baseMode":"UNKNOWN", "met_equivalent":"UNKNOWN", "kgCo2PerKm": 0}   
+    {"value":"walk_run", "base_mode": "WALKING", "met": {"ALL": {"range": [0, -1], "mets": 3}},
+      "baseMode":"WALKING", "kgCo2PerKm": 0},
+    {"value":"auto_driver", "base_mode": "CAR",
+      "baseMode":"CAR", "met_equivalent":"IN_VEHICLE", "kgCo2PerKm": 0.22031},
+    {"value":"auto_passenger", "base_mode": "CAR", "passengers": 2,
+      "baseMode":"CAR", "met_equivalent":"IN_VEHICLE", "kgCo2PerKm": 0.110155},
+    {"value":"e-bike", "base_mode": "E_BIKE",
+      "baseMode":"E_BIKE", "met": {"ALL": {"range": [0, -1], "mets": 5}}, "kgCo2PerKm": 0.00728},
+    {"value":"bike", "base_mode": "BICYCLING",
+      "baseMode":"BICYCLING", "met": {"ALL": {"range": [0, -1], "mets": 6}}, "kgCo2PerKm": 0},
+    {"value":"taxi", "base_mode": "TAXI",
+      "baseMode":"TAXI", "met_equivalent":"IN_VEHICLE", "kgCo2PerKm": 0.30741},
+    {"value":"bike_scooter_share", "base_mode": "BICYCLING", "footprint": {"electric": {"wh_per_km": 8.39, "wh_per_trip": 2.05 }},
```

Comment:
The `kgCo2PerKm` for `bike_scooter_share` was `0.00447`, which is exactly half of what the default e_scooter `kgCo2PerKm` was.

So it seems like bikeshare was assumed to be 0 (rebalancing was ignored) and then an average was taken of bikeshare and scootershare.

For the new `footprint` field, I have done the same – took half of what the new scootershare footprint is (electric @ 16.78 Wh / km + 4.1 Wh / trip)